requirements/dev.txt

```diff
@@ -14,3 +14,6 @@ pytest < 8
 pytest-order < 2
 pytest-xdist < 4
 pyyaml
+pycountry
```

Comment:
should probably pin this. If country names we're referencing change, it can cause issues (in euro-calliope, it was North Macedonia)

Reply:
I am using fuzzy search to get around those cases. I also found the Macedonia case, but the fuzzy search finds it OK. But pinning is a good idea.